data/data/openstack/bootstrap/main.tf

```diff
@@ -52,5 +52,5 @@ resource "openstack_networking_floatingip_v2" "bootstrap_fip" {
   port_id     = openstack_networking_port_v2.bootstrap_port.id
   tags        = ["openshiftClusterID=${var.cluster_id}"]
 
-  depends_on = ["openstack_compute_instance_v2.bootstrap"]
+  depends_on = ["openstack_compute_instance_v2.bootstrap", "openstack_networking_port_v2.bootstrap_port"]
```

Comment:
According to the [terraform documentation](https://learn.hashicorp.com/terraform/getting-started/dependencies.html) there's already an implicit dependency on `openstack_networking_port_v2.bootstrap_port` because of line 52. It's not clear to me how making it an explicit dependency is going to solve the issue.

Reply:
agree, this change doesn't make much sence, since it's just an internal terraform dependency.
Now I'm going to implement the fix by adding a port data source.